gnd/src/main/java/com/google/android/gnd/persistence/local/room/Coordinates.java

```diff
@@ -17,40 +17,24 @@
 package com.google.android.gnd.persistence.local.room;
 
 import androidx.annotation.NonNull;
-import androidx.room.ColumnInfo;
-import com.google.auto.value.AutoValue;
-import com.google.auto.value.AutoValue.CopyAnnotations;
+import com.google.android.gnd.vo.Point;
 
-/** Represents a lat/lng coordinate in FeatureEdit instances. */
-@AutoValue
-public abstract class Coordinates {
-  @CopyAnnotations
-  @NonNull
-  @ColumnInfo(name = "latitude")
-  public abstract double getLatitude();
-
-  @CopyAnnotations
-  @NonNull
-  @ColumnInfo(name = "longitude")
-  public abstract double getLongitude();
-
-  // Auto-generated boilerplate:
-
-  public static Coordinates create(double latitude, double longitude) {
-    return builder().setLatitude(latitude).setLongitude(longitude).build();
-  }
-
-  public static Builder builder() {
-    return new AutoValue_Coordinates.Builder();
-  }
-
-  @AutoValue.Builder
-  public abstract static class Builder {
+/**
+ * Defines how Room persists coordinates in the local db. By default, Room uses the name of object
+ * fields and their respective types to determine database column names and types.
+ */
+public class Coordinates {
 
-    public abstract Builder setLatitude(double newLatitude);
+  @NonNull public double latitude;
```

Comment:
These classes exist solely as a mapping between our domain objects and the database, and are meant to be referenced only by clients of the Daos saving and retrieving data from the local db. I could change these back to `@AutoValue` classes as they were before, but since we we're using these classes to mirror the db representation by definition, I'm not sure how much value the extra boilerplate adds. Thoughts?

Reply:
AutoValue might be helpful if we ever need to put these in, say, a container where `hashCode` and `equals` are useful. And using getters/setters instead of direct field access would help in that it allows us to do things like switch to AutoValue without changing any of the callers.

Reply:
Sounds like these are hypothetical cases that are unlikely to occur. I could see some benefit in being consistent with other value objects, but it does seem like a lot necessary boilerplate, especially if we use @AutoValue. I'll convert these back for now though to be able to move forward.